### DISCUSSION TITLE
Tokens for the incentives in Technical brainstorms

### DISCUSSION THREAD

**Discussion by sergfeldman** [pavlovcik/uad-ui#186] [Voting]
Please
1) Vote for the token (uCR or UBQ or USDC) that you would like to get for proposals in the Technical brainstorm.
2) Add a comment on how much in which token would you like to get for 1 USDC
1 USDC = ? uCR
1 USDC = ? UBQ

**Comment by 0x4007** [comment]
The organization is incentivized to give out rewards in NOT cash equivalents (USDC/DAI etc) as it is the most rare resource of the three categories (cash/credits/governance)

This is always a struggle as I feel like no standard exchange rate has been established across the board for operations compensation. However I have done `1 USDC = 1.5 uCR` and given UBQ as a cash match `1 USDC` grants `1 UBQ` but I feel like some strategy should be formalized for UBQ.
